spec.bs

```diff
@@ -1022,6 +1086,38 @@ Note: Previously, the `payment` extension allowed for the creation of
       :  <dfn>instrument</dfn> member
       :: The instrument details that were displayed to the user. Only used at authentication time; not registration.
 
+      :  <dfn>browserBoundPubKeyCredParams</dfn> member
+      :: The list of allowed types of credential restricting the types of
+         cryptographic algorithms used for the browser bound key.
+
+    </dl>
+
+:  Client extension outputs
+::
+    <xmp class="idl">
+    partial dictionary AuthenticationExtensionsClientOutputs {
+      AuthenticationExtensionsPaymentOutputs payment;
+    };
+
+    dictionary AuthenticationExtensionsPaymentOutputs {
+      AuthenticationExtensionPaymentBrowserBoundSignature browserBoundSignature;
+    };
+
+    dictionary AuthenticationExtensionPaymentBrowserBoundSignature {
+      required ArrayBuffer signatureOutput;
```

Comment:
Bikeshedding, but WebAuthn just calls their signature arraybuffers `signature`, so I suggest we do the same (i.e. drop `Output`).